images/grafana/Dockerfile

```diff
@@ -1,7 +1,50 @@
+FROM debian:stretch-slim
```

Comment:
I used this Dockerfile from upstream because I need to add one package in container. I just removed few unneeded lines.